libr/core/cbin.c

```diff
@@ -179,6 +179,11 @@ static void _print_strings(RCore *r, RList *list, int mode, int va) {
 		r_flag_space_set (r->flags, "strings");
 		r_cons_break_push (NULL, NULL);
 	}
+	if (IS_MODE_NORMAL (mode)) {
+		r_cons_printf ("[Strings]\n");
+		r_cons_printf ("Num Vaddr      Paddr      Len Size "
```

Comment:
just put this on one line. Though I'm generally in favour of splitting long lines, I think in this case it just reduces "greppability" and readability.